### ISSUE TITLE
DCB file signed 16 bit integer

### ISSUE THREAD

**Issue by Tecuma1** [cantools/cantools#614]
Hi everybody,

I am trying to decode can data using a dbc file. 
I know the value that I should be getting is -0.2
The raw can data: FF FE

According to the datasheet of the BMS that I am using it uses a 16 bit signed integer with a 0.1 scale.
If I write my DBC file like this:

 SG_ Current : 0|16@1- (0.1,0) [-3276.8|3276.7] "A" Vector
 I am getting -25.4

If I write it like this:
 SG_ Current : 8|8 @1- (0.1,0) [-3276.8|3276.7] "A" Vector
I am getting the -0.2

Basically with the second approach I am ignoring the FF part of the dataframe. (Which should be 0)
For now this approach works but if the value surpasses 127 I am in trouble.
For some reason the cantools library looks at the data differently. But I am not sure what to change.

**Comment by juleq** [comment]
Check the endianness. -247*0.1 is the result when u swap bytes...
